tests/e2e/network/multiple_standard_lb.go

```diff
@@ -178,8 +178,87 @@ var _ = Describe("Ensure LoadBalancer", Label(utils.TestSuiteLabelMultiSLB), fun
 		Expect(lbNames.Has("lb-1")).To(BeFalse())
 		Expect(lbNames.Has(clusterName)).To(BeTrue())
 	})
+
+	It("should arrange local services", func() {
+		By("Creating a local service")
+		svc := utils.CreateLoadBalancerServiceManifest(testServiceName, nil, labels, ns.Name, ports)
+		svc.Spec.ExternalTrafficPolicy = v1.ServiceExternalTrafficPolicyTypeLocal
+		_, err := cs.CoreV1().Services(ns.Name).Create(context.TODO(), svc, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		ips, err := utils.WaitServiceExposureAndValidateConnectivity(cs, tc.IPFamily, ns.Name, testServiceName, []string{})
+		Expect(err).NotTo(HaveOccurred())
+		Expect(len(ips)).NotTo(BeZero())
+
+		By("Checking the node count in the local service backend pool to equal 5")
+		clusterName := os.Getenv("CLUSTER_NAME")
+		nodes, err := utils.GetAgentNodes(cs)
+		Expect(err).NotTo(HaveOccurred())
+		expectedBPName := fmt.Sprintf("%s-%s", svc.Namespace, svc.Name)
+		err = checkNodeCountInBackendPoolByServiceIPs(tc, clusterName, expectedBPName, ips, len(nodes))
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Scaling the deployment to 3 replicas and then to 1")
+		deployment.Spec.Replicas = pointer.Int32(3)
+		_, err = cs.AppsV1().Deployments(ns.Name).Update(context.Background(), deployment, metav1.UpdateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		deployment.Spec.Replicas = pointer.Int32(1)
+		_, err = cs.AppsV1().Deployments(ns.Name).Update(context.Background(), deployment, metav1.UpdateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		err = wait.PollUntilContextTimeout(context.Background(), 5*time.Second, 5*time.Minute, false, func(ctx context.Context) (bool, error) {
+			if err := checkNodeCountInBackendPoolByServiceIPs(tc, clusterName, expectedBPName, ips, 1); err != nil {
+				if strings.Contains(err.Error(), "expected node count") {
+					utils.Logf("Waiting for the node count in the backend pool to equal 1")
+					return false, nil
+				}
+				return false, err
+			}
+			return true, nil
+		})
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Scaling the deployment to 5")
+		deployment.Spec.Replicas = pointer.Int32(5)
+		_, err = cs.AppsV1().Deployments(ns.Name).Update(context.Background(), deployment, metav1.UpdateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		err = wait.PollUntilContextTimeout(context.Background(), 5*time.Second, 5*time.Minute, false, func(ctx context.Context) (bool, error) {
+			if err := checkNodeCountInBackendPoolByServiceIPs(tc, clusterName, expectedBPName, ips, len(nodes)); err != nil {
```

Comment:
so len(nodes) <= 5?

Reply:
It can be any number. I have a wrong log in L192.